configs/nextflow.base.config

```diff
@@ -108,13 +112,13 @@ process {
 		container = "${params.containerDir}/bwakit.sif"
 		cpus = 16
 		memory = '20 GB'
-		publishDir = [ path: "${params.outdir}/${params.speciesDir}/bwa", mode: 'copy', overwrite: true ]
```

Comment:
Not sure what happened here... Guess it is legit, but so the publishdir was just moved?

Reply:
I was publishing the wrong bam file. `bwa` is run on it's own assembly for qc purposes as well as against a reference genome to check variants